proofs/eo/cpc/rules/Rewrites.eo

```diff
@@ -7,1972 +7,3 @@
 (include "../theories/Strings.eo")
 (include "../theories/Sets.eo")
 
-(declare-rule arith-mul-one ((@T0 Type) (@T1 Type) (t1 @T0 :list) (s1 @T1 :list))
```

Comment:
There was an issue in the auto-generation of this file. You should make sure to compile cvc5 and then rerun the `install-rare-rewrites` script.